app/Console/Commands/CheckForOpenCFPs.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class CheckForOpenCFPs extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'symposium:checkForOpenCFPs';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send out notification for open CFPs';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        //
```

Comment:
OK, so in here it's gonna be: Conference::approved()->cfpOpensToday()->each( queue notification for all users who have opted in), right?

Reply:
Maybe just two separate scopes

Conference::approved()->unNotified()